syft/formats/common/spdxhelpers/license.go

```diff
@@ -22,16 +22,24 @@ func License(p pkg.Package) string {
 	}
 
 	// take all licenses and assume an AND expression; for information about license expressions see https://spdx.github.io/spdx-spec/appendix-IV-SPDX-license-expressions/
-	var parsedLicenses []string
-	for _, l := range p.Licenses {
-		if value, exists := spdxlicense.ID(l); exists {
-			parsedLicenses = append(parsedLicenses, value)
-		}
-	}
+	parsedLicenses := ParseLicenses(p.Licenses)
 
 	if len(parsedLicenses) == 0 {
 		return NOASSERTION
 	}
 
 	return strings.Join(parsedLicenses, " AND ")
 }
+
+func ParseLicenses(raw []string) (parsedLicenses []string) {
```

Comment:
I don't think this needs to be exported (e.g. it can start with lowercase `parseLicenses`)

Reply:
Easy enough. Done.